src/xercesc/util/NetAccessors/WinSock/BinHTTPURLInputStream.cpp

```diff
@@ -196,32 +196,53 @@ void BinHTTPURLInputStream::Initialize(MemoryManager* const manager)
 #ifdef WITH_IPV6
       gWinsockLib = LoadLibraryA("WS2_32");
 #else
-      gWinsockLib = LoadLibraryA("WSOCK32");
+      gWinsockLib = LoadLibraryW(L"WSOCK32");
```

Comment:
Isn't this change also affecting other platforms?

Reply:
You are correct, Luca. This is an issue, need to fix it by using WINCE macro.
#ifdef WINCE
    #ifdef WITH_IPV6
            gWinsockLib = LoadLibraryW(L"WS2_32");
    #else
            gWinsockLib = LoadLibraryW(L"WSOCK32");
    #endif
#else
    #ifdef WITH_IPV6
          gWinsockLib = LoadLibraryA("WS2_32");
    #else
          gWinsockLib = LoadLibraryW(L"WSOCK32");
    #endif
#endif